source/Armament.h

```diff
@@ -63,6 +66,18 @@ class Armament {
 
 	// Access the array of weapon hardpoints.
 	const std::vector<Hardpoint> &Get() const;
+	const std::vector<Hardpoint *> AllWeaponsNoAM() const;
```

Comment:
how about NonAMWeapons() instead for the name?

Reply:
`OffensiveWeapons` maybe?